test/utils.ts

```diff
@@ -26,14 +27,9 @@ export abstract class Utils {
         version + '/rest';
   }
 
-  static loadApi(
-      google: GoogleApis, name: string, version: string, options, cb) {
-    if (typeof options === 'function') {
-      cb = options;
-      options = {};
-    }
-    return google.discoverAPI(
-        Utils.getDiscoveryUrl(name, version), options, cb);
+  static async loadApi(
+      google: GoogleApis, name: string, version: string, options = {}) {
+    return google.discoverAPI(Utils.getDiscoveryUrl(name, version), options);
```

Comment:
should we await for the result here, then return?

Reply:
nah, don't need to.  I actually deleted the `async` from the fn, because it just returns a promise (which is awaitable) directly.